guides/v2.1/cloud/env/variables-deploy.md

```diff
@@ -424,15 +424,29 @@ stage:
 -  **Default**—`true`
 -  **Version**—Magento 2.1.4 and later
 
-On deployment, replace Magento base URLs in the database with project URLs. This is useful for local development, where base URLs are set up for your local environment. When you deploy to a Cloud environment, we change the URLs so you can access your storefront and Magento Admin using project URLs.
+On deployment, replace Magento base URLs in the database the project URLs specified in the `MAGENTO_CLOUD_ROUTES` variable. This is useful for local development, where base URLs are set up for your local environment. 
+When you deploy to a Cloud environment, we change the URLs so you can access your storefront and Magento Admin using project URLs.
+
+To update URLs when deploying to staging or production environments,                             use the `FORCE_UPDATE_URLS` variable. 
 
 ```yaml
 stage:
   deploy:
     UPDATE_URLS: false
 ```
 
-You should set this variable to `false` _only_ in Staging or Production environments, where the base URLs cannot change. For Pro, we already set this to `false` for you.
+### `FORCE_UPDATE_URLS`
```

Comment:
All of these variables are in alphabetical order.

Reply:
So I know that -- I thought this one is so closely related to the UPDATE_URLS one that it made sense to put them together.  It's a trade off.  I can move to the correct alphabetical location if you think that's more important than proximity to related variable.

Reply:
in reference material, proximity is not something I consider. I don't think that I would go to the U if I were looking for F. Although, my instinct is CMD+F in this case,  the right navigation grabs the alphabetic attention. so that would be the anomaly.